### PR TITLE
Document how to configure when behind a reverse proxy

### PR DESCRIPTION
**Pull request by bliuredhat**
JIRA: PDC-1520

### CODE REVIEW

File: docs/source/deployment.rst [hunk 1]
```diff
@@ -53,3 +53,9 @@ Config apache
 -------------
 
 replace ``PDC_HOSTNAME`` with server's hostname in ``/etc/httpd/conf.d/pdc.conf``
+
+
+Enabled proxy
+-------------
+
+Add ``USE_X_FORWARDED_HOST = True`` in `local_setting.py` file.
```
Comment (lubomir) [inline docs/source/deployment.rst]: The file should be called `settings_local.py`, right?

File: docs/source/deployment.rst [hunk 2]
```diff
@@ -53,3 +53,9 @@ Config apache
 -------------
 
 replace ``PDC_HOSTNAME`` with server's hostname in ``/etc/httpd/conf.d/pdc.conf``
+
+
+Enabled proxy
```
Comment (sochotnicky) [inline docs/source/deployment.rst]: Maybe change to "Running PDC behind reverse proxy"

File: docs/source/deployment.rst (hunk 1 above)
Comment (sochotnicky) [inline docs/source/deployment.rst]: I'd add a 1-2 sentences explaining why it has to be done. Something like - "To make sure documentation links work correctly when PDC is running behind proxy, add ...."

Comment (bliuredhat) [inline docs/source/deployment.rst]: @lubomir , yes

Comment (lubomir) [inline docs/source/deployment.rst]: Also a link to Django documentation might be useful: https://docs.djangoproject.com/en/1.9/ref/settings/#use-x-forwarded-host

File: docs/source/deployment.rst (hunk 2 above)
Comment (bliuredhat) [inline docs/source/deployment.rst]: @sochotnicky; cool~

**Comment by ycheng-aa** [comment]
:+1:

**Comment by simozhan** [comment]
looks good

**Pull request merged**